policy.go

```diff
@@ -64,9 +64,16 @@ type dynamicPolicyComputeParams struct {
 	policyCount uint64
 }
 
+type policyOrDataNode struct {
+	Next    uint32
```

Comment:
this probably merits a comment about how it's defined, is it absolute or relative to the current node?